src/main/java/app/service/impl/FruitParserImplements.java

```diff
@@ -0,0 +1,23 @@
+package app.service.impl;
+
+import app.model.SupplyFruit;
+import app.service.FruitParser;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+public class FruitParserImplements implements FruitParser {
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
in LocalDate this format is used by  default. You don't need to use it

Reply:
Sure. Done.